network.tf

```
@@ -85,18 +49,21 @@ resource "aws_route_table" "without-peer" {
     cidr_block = "0.0.0.0/0"
     gateway_id = aws_internet_gateway.main.id
   }
+  tags = local.common_tags
+}
 
-  tags = {
-    Name            = "${var.vpc_name}-${var.env}"
-    RawName         = var.vpc_name
-    OwnerList       = var.owner
-    EnvironmentList = var.env
-    EndDate         = var.end_date
-    ProjectList     = var.project
-    DeploymentType  = var.deployment_type
+resource "aws_route_table" "connection_to_nat_gatway" {
```

Can we name it something that identifies this as specific to the private subnet? 

e.g 

```suggestion
resource "aws_route_table" "private_subnet_connection_to_nat_gatway" {
```

---

@morrismukiri updated as requested. Please review again.